src/erlcloud_iam.erl

```diff
@@ -704,6 +706,38 @@ get_credential_report(Config) ->
                    {"Content", content, "String"}],
     iam_query(Config, "GetCredentialReport", [], ItemPath, DataTypeDef).
 
+simulate_principal_policy(PolicySourceArn, ActionNames) ->
+    simulate_principal_policy(PolicySourceArn, ActionNames, default_config()).
+
+simulate_principal_policy(PolicySourceArn, ActionNames, Config)
+  when is_list(PolicySourceArn) andalso is_list(ActionNames) ->
+    ItemPath = "/SimulatePrincipalPolicyResponse/SimulatePrincipalPolicyResult",
+    Params = [{"PolicySourceArn", PolicySourceArn} |
+              erlcloud_util:encode_list("ActionNames", ActionNames)],
+    case iam_query_all(Config, "SimulatePrincipalPolicy", Params,
+                       ItemPath, data_type("EvaluationResults")) of
+        {ok, [Res]} ->
```

Comment:
even single Get* funs return `{ok, [proplist()]}` in the module. 
please remove this and be consistent with the rest otherwise it'll cause confusion.